packages/paste-icons/src/AgentIcon.tsx

```diff
@@ -24,6 +24,7 @@ const AgentIcon = React.forwardRef<HTMLElement, AgentIconProps>(
         <svg
           role="img"
           aria-hidden={decorative}
+          xmlns="http://www.w3.org/2000/svg"
```

Comment:
Question:  why did you add this to all the icons?

Reply:
I just ran convert all, so that the updated svg's of existing icons are converted to new React components. `convert-new` only does missing react components. 

I actually assumed it was also adding it to the new icons, but it isn't. So that's weird... 🤷🏻‍♂️

Reply:
yeah, I had the same question bc it was happening in my branch.

I removed xmlns from the SVG export on the new icons because I thought they were breaking the command somehow. But these SVGs weren't changed at all so I don't know why it's doing it now.

Reply:
... weird, nothing changed in svg-to-react recently

Reply:
It's not needed, but not a big deal either. Just adds extra bytes needlessly to the icons. Won't call it a blocker. I wonder if it crept up due to not pinning deps in SVG-to-React